observer/stream.go

```diff
@@ -87,7 +87,7 @@ func (s *Stream) loadBlock(c chan<- *blockatlas.Block, num int64) {
 	s.semaphore.Acquire()
 	defer s.semaphore.Release()
 
-	block, err := s.BlockAPI.GetBlockByNumber(num)
+	block, err := retry(3, time.Second*1, s.BlockAPI.GetBlockByNumber, num)
```

Comment:
can we do 5 retries? it would do 1, then wait 5 sec, then 15 sec and then 30 sec and 60? any examples on how that works.

would be great to log errors on this, if some block failed

Reply:
done!